### ISSUE TITLE
Prepare script for tutorial video about different types of files in the RADx Data Hub

### ISSUE THREAD

**Issue by matthewhorridge** [bmir-radx/radx-project#117]

**Comment by matthewhorridge** [comment]
@marcosmro I've placed this under 1.2B is this the correct place for it?  I know B tasks are not our tasks.

**Comment by marcosmro** [comment]
@matthewhorridge Yes, that's the right place for it, thanks. Even though RENCI is the team responsible for 'B' tasks, we are also listed under 1.2B and are expected to do some work on it, so it's fine.
